KsApi/queries/CommentsQueries.swift

```diff
@@ -28,3 +42,38 @@ public func commentsQuery(
     ]
   ) +| []
 }
+
+/**
+ Constructs a query to fetch a `Comment`'s replies. Accepts an optional cursor to page in the
+ most replies before that cursor from most recent to oldest.
+
+ - parameter id: A parent `Comment`'s ID to fetch replies for.
+ - parameter last: An optional limit parameter, defaulted to `Query.defaultPaginationCount`.
+ - parameter before: An optional cursor to fetch the most recent replies before.
+
+ - returns: A `NonEmptySet<Query>`.
+ */
+public func commentRepliesQuery(
+  withCommentId id: String,
+  last: Int = Query.defaultPaginationCount,
+  before cursor: String? = nil
+) -> NonEmptySet<Query> {
+  let args = Set([cursor.flatMap(QueryArg<Never>.before), .last(last)].compact())
+
+  return Query.comment(
+    id: id,
+    GraphComment.baseQueryProperties.op(
```

Comment:
is `op` like a query in another query? I noticed it's used above as well to get the replies from within each `Comment` after the `baseProperties` get called.

Reply:
It allows us to concatenate the queries: https://github.com/kickstarter/Kickstarter-Prelude/blob/master/Prelude/NonEmpty.swift#L92